.github/ISSUE_TEMPLATE.md

```diff
@@ -2,7 +2,7 @@
 
 This issue tracker is only for technical issues related to Dogecoin Core.
 
-For general questions about Dogecoin please use one of the various communities, e.g. Reddit, IRC, Discord, etc.
+For general questions about Dogecoin or wallet recovery please use one of the various communities, e.g. Reddit, IRC, Discord, etc.
```

Comment:
```suggestion
For general questions about Dogecoin or wallet recovery please use one of the various communities:

* [Dogeducation on reddit](https://www.reddit.com/r/dogeducation/)
* [Discord](https://discord.com/invite/dogecoin)
```

Might want to expand that further, thinking about it, will leave the other devs to comment.

Reply:
I skimmed the sidebar links in the dogecoin reddit, and those two links seem far and away the most useful. IRC was my third choice, but that seems better for dev than wallet support.